requirements.txt

```diff
@@ -1,6 +1,5 @@
---extra-index-url https://dl.cloudsmith.io/public/borgmatic-collective/borgmatic/python/simple/
+--extra-index-url https://dl.cloudsmith.io/public/modem7/wheels/python/simple/
```

Comment:
Another instance :)

Reply:
Both done in https://github.com/borgmatic-collective/docker-borgmatic/pull/303/commits/ea6d942f3588e4b61843f4b9b9e5a81ec048288f